core-bundle/tests/Controller/FrontendModule/FrontendModuleControllerTest.php

```diff
@@ -158,10 +190,14 @@ private function mockContainerWithFrameworkTemplate(string $templateName): Conta
             ->willReturn(new FrontendTemplate($templateName))
         ;
 
-        $container = new ContainerBuilder();
-        $container->set('contao.framework', $framework);
-        $container->set('contao.routing.scope_matcher', $this->mockScopeMatcher());
+        $this->container->set('contao.framework', $framework);
+        $this->container->set('contao.routing.scope_matcher', $this->mockScopeMatcher());
+
+        return $this->container;
+    }
 
-        return $container;
+    private function getModuleModel(): ModuleModel
+    {
+        return (new \ReflectionClass(ModuleModel::class))->newInstanceWithoutConstructor();
```

Comment:
Why can we no longer use `new ModuleModel()`?

Reply:
Creating an instance of `Model` will extract DCA information and fail miserably without the framework booted.